### PR TITLE
Allow adding bundler plugin execs in local config.

### PR DESCRIPTION
**Pull request by bobthecow**
```fish
set bundler_plugin_execs jekyll mustache
set fish_plugins bundler
```

### CODE REVIEW

**Comment by bpinto** [comment]
Add more execs instead? These are not user specific executables, so the more execs we support the best for everyone.

**Comment by bobthecow** [comment]
Jekyll, yeah, but Mustache isn't bundler/ruby-specific... You can install a mustache executable via npm, and any number of other package managers. Even if there are a lot that should be included, there still should be a mechanism for explicitly setting ones that aren't universally applicable.

**Comment by bpinto** [comment]
If mustache is not bundler/ruby-specific why run it with `bundle exec`?

Do we still need to run commands with `bundle exec` now that binstub is the preferred way to run these commands?

**Comment by bobthecow** [comment]
It's not *always* bundler/ruby-specific. In my case, there's an executable called `mustache` that I installed via rubygems. But there's also an executable in the mustache.js npm package. Hijacking that would be rude :)

File: plugins/bundler/bundler.load [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # These methods override the default calls to append `bundle exec` if this gem
 # is available within the context of bundler.
 #
+# You can add more bundled executables in your config.fish:
+#
+#     set bundler_plugin_execs jekyll mustache
```
Comment (bpinto) [inline plugins/bundler/bundler.load]: Does it need to be `set -g`?

**Comment by bpinto** [comment]
:P Okay.

File: plugins/bundler/bundler.load (hunk 1 above)
Comment (bobthecow) [inline plugins/bundler/bundler.load]: Doesn't seem to? I usually write that, but I realized when adding the instructions for this than nothing in my config.fish is `set -g`.

**Comment by bucaran** [comment]
Errors are fixed now. Rebase please :)

**Comment by bobthecow** [comment]
No problem. Rebased.

**Comment by bucaran** [comment]
:+1:

**Pull request merged**